src/main/java/sirius/biz/storage/layer3/ExtractArchiveJob.java

```diff
@@ -176,7 +178,7 @@ private void handleExtractedFile(ExtractedFile extractedFile,
         switch (result) {
             case CREATED -> process.addTiming(NLS.get("ExtractArchiveJob.fileCreated"), watch.elapsedMillis());
             case UPDATED -> process.addTiming(NLS.get("ExtractArchiveJob.fileOverwritten"), watch.elapsedMillis());
-            case SKIPPED -> process.addTiming(NLS.get("ExtractArchiveJob.fileSkipped"), watch.elapsedMillis());
+            case SKIPPED -> process.addTiming(FILE_SKIPPED_MESSAGE, watch.elapsedMillis());
```

Comment:
I think these timings should use the key directly instead of translating them so they are dynamically translated in the UI